### PR TITLE
3DCursor: Change width of toggle button in UI

### PR DESCRIPTION
**Pull request by parthmehta209**

### CODE REVIEW

**Comment by parthmehta209** [comment]
@rahul27 minor ui change for fitting toggle button text.

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <dimen name="iodevice_element_margin">3dp</dimen>
     <dimen name="cursor_list_io_dev_min_width">100dp</dimen>
     <dimen name="cursor_list_text_box_padding">6dp</dimen>
-    <dimen name="toggle_button_width">68dp</dimen>
+    <dimen name="toggle_button_width">77dp</dimen>
```
Comment (rahul27) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/res/values/dimens.xml]: Should be fine as long as it shows up correctly on the Note and the GS.

**Comment by thomasflynn** [comment]
is this good to go in?

**Comment by rahul27** [comment]
Was waiting for @parthmehta209 to test this against the Note4. Let me do it quickly and will let you know.

**Comment by thomasflynn** [comment]
don't worry, i wasn't rushing.  i just hadn't seen any traffic on this in a while.  was curious about status.

**Comment by rahul27** [comment]
Verified. Should be good to go.

**Comment by thomasflynn** [comment]
thanks.

**Comment by parthmehta209** [comment]
Havent tested on the note4 yet. Its fine on the s6.

**Pull request merged**